thirdparty/scripts/build_arrow.sh

```diff
@@ -49,7 +49,7 @@ build_arrow() {
     if [[ -d $TP_DIR/build/arrow ]]; then
       rm -rf $TP_DIR/build/arrow
     fi
-    git clone -q https://github.com/apache/arrow.git "$TP_DIR/build/arrow"
+    git clone -q https://github.com/pcmoritz/arrow.git "$TP_DIR/build/arrow"
```

Comment:
revert?

Reply:
Thanks, fixed it (it had to wait till the arrow PR was merged)